allennlp/training/metrics/fbeta_multi_label_measure.py

```diff
@@ -90,8 +90,8 @@ def __call__(
         predictions : `torch.Tensor`, required.
             A tensor of predictions of shape (batch_size, ..., num_classes).
         gold_labels : `torch.Tensor`, required.
-            A tensor of integer class label of shape (batch_size, ...). It must be the same
-            shape as the `predictions` tensor without the `num_classes` dimension.
+            A tensor of integer class label of shape (batch_size, ..., num_classes).
+            It must be the same shape as the `predictions` tensor.
```

Comment:
How can that be? The integer values are indices into the `num_classes` dimension, no?